### ISSUE TITLE
[Transfer proposal] Move css-visor to webpack-contrib

### ISSUE THREAD

**Issue by zikaari** [webpack-contrib/organization#16]
I just released css-visor, which aims to fill gaps left by style-loader (like having sourcemaps and no FOUC at the same time).

It lives at https://github.com/chipto/css-visor and I would much prefer it is in a lot of good hands.

Even though it's just a baby, I have a strong feeling about it's bright side. Please give it a shot and share what you feel.

Also I don't mind if you'd like to change the name to something else.

**Comment by zikaari** [comment]
Any green light or initial thoughts on this?

See this PR for more context: https://github.com/facebookincubator/create-react-app/pull/3202

cc [@TheLarkInn, @evilebottnawi, ...otherContribs]

**Comment by michael-ciniawsky** [comment]
we definitely need to define a process for this first (e.g requirements, how find consensus etc). The fact is that we don't currently have one 😥, so I can't comment on your particular transfer request (yet). Tracking issue is #19 and I'm going to close this issue for now to keep focus. Please feel free to open a new issue (transfer request) once a process for this is in place. Sry for the inconvenience here
